codegen/java-gen/src/main/scala/com/lightbend/akkasls/codegen/java/ActionServiceSourceGenerator.scala

```diff
@@ -241,7 +241,7 @@ object ActionServiceSourceGenerator {
       val inputTypeFullName = cmd.inputType.fullName
 
       s"""|case "$methodName":
-          |  return action()
+          |  return (Source<Action.Effect<?>, NotUsed>)(Object) action()
```

Comment:
my Java parser is in for repair; does it first cast to Object followed by another cast to Source?

Reply:
Yes, since the Java doesn't understand variance/Java `Source` is invariant. I don't think there is any other way to do it. We _know_ it is safe though.

Reply:
I trust you ™️ 
Shall we add some supress warning annotation?

Reply:
Yes, we should 👍